java/core/lance-jni/src/error.rs

```diff
@@ -41,8 +41,8 @@ impl JavaException {
 pub enum Error {
     #[snafu(display("JNI error: {}", message))]
     Jni { message: String },
-    #[snafu(display("Invalid argument: {}", message))]
-    InvalidArgument { message: String },
+    #[snafu(display("Invalid argument: {}, location: {}", message, location))]
+    InvalidArgument { message: String, location: Location },
```

Comment:
why do we need Location in the error msg?

Reply:
this location tells us which line generated the error. This is how we get a "stack track" when an error happens